README.md

```diff
@@ -139,3 +139,9 @@ This project is licensed under the MIT License - see the [LICENSE](LICENSE) file
 ## Acknowledgments
 - The California housing dataset is sourced from scikit-learn.
 - SHAP and LIME libraries are used for model interpretability.
+
+## Inspiration
+
```

Comment:
no. I would not add this. the inspiration should be in the structure level of the readme. nothing regarding the content.

Reply:
Sorry, I got you wrong. I've uploaded the updated version of the readme

Reply:
please mark resolve once finish so I could approve the PR